### PR TITLE
ci(seeder): github actions

### PR DESCRIPTION
**Pull request by moonlitgrace**

### CODE REVIEW

File: .github/workflows/seeder.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      name: Node ${{ matrix.node-version }} on ${{ matrix }}
+      steps:
+        - name: Checkout Seeder
+          uses: actions/checkout@4
+        - name: Setup Node
+          uses: actions/setup-node@v3
+          with:
+            node-version: ${{ matrix.node-version }}
+            cache: "npm"
+        - run: npm i
```
Comment (baseplate-admin) [inline .github/workflows/seeder.yml:36]: npm ci

File: .github/workflows/seeder.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        node-version:
+          - 22.x
```
Comment (baseplate-admin) [inline .github/workflows/seeder.yml:26]: Use latest

File: .github/workflows/seeder.yml [hunk 3]
```diff
@@ -22,18 +26,14 @@ jobs:
...
         uses: actions/checkout@v4
-        with:
-          path: ./seeder
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
           cache: "npm"
-        working-directory: ./seeder
+          cache-dependency-path: seeder/package-lock.json
```
Comment (baseplate-admin) [inline .github/workflows/seeder.yml:34]: Very unnecessary

Comment (moonlitgrace) [reply .github/workflows/seeder.yml]: https://github.com/coreproject-moe/CoreProject/actions/runs/10337853761/job/28615192230?pr=3962#step:3:15
https://dev.to/shofol/run-your-github-actions-jobs-from-a-specific-directory-1i9e#comment-1mo5m

**Pull request merged**